docs/requests.md

```diff
@@ -138,7 +146,7 @@ For example, you can get the file name and the contents with:
 
 ```python
 form = await request.form()
-filename = form["upload_file"].filename
+filename = form["upload_file"].filename # or form.get("upload_file")
```

Comment:
I understand the goal of adding a second way of doing it, but if a meaningful example is wanted here, it would be cool to be a separate one 🤔 
```suggestion
filename = form["upload_file"].filename
```